guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,20 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response (static page) that never gets expired.
+To achieve that, we can use `http_cache_forever` helper and by doing so browser
+and proxies should cache it indefinitely.
+
+```ruby
+class HomeController < ApplicationController
+  def index
+    http_cache_forever(public: params[:public], version: params[:version] || 'v1') do
```

Comment:
https://github.com/prathamesh-sonpatki/rails/blob/master/actionpack/lib/action_controller/metal/conditional_get.rb#L259

The methods accepts only one argument - `public`, Also I don't think the public value will come from the `params`, it will be server determined value.